docs/pages/docs/05-distributed-analytics/compute-centrality.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Centrality
+tags: [analytics]
+summary: "This page introduces the computation of centrality in a graph."
+sidebar: documentation_sidebar
+permalink: /docs/distributed-analytics/compute-centrality
+folder: docs
+---
+
+What is the most important instance in a graph? 
+More importantly, how do we measure importance in a graph?
+
+Indicators of centrality identify the most important vertices within a graph. 
+[Wikipedia](https://en.wikipedia.org/wiki/Centrality)
+
+Currently, Graql supports two algorithms for computing centrality: 
+- Degree 
+- K-Core
+
+## Degree
+
+The degree of an instance gives the number of other instances directly connected to the instance. 
+For an entity, such as a `person`, this means the number of connections, 
+for example instances of a `marriage` relationship.
+The `person` with the highest degree, or the greatest number of marriages, is arguably the most "interesting"
+([Glynn Wolfe](https://en.wikipedia.org/wiki/Glynn_Wolfe)).
+
+In analyics, the `degree` algorithm computes how many arrows (edges) there are attached to instances in the graph. 
+A map is returned containing degree and instances with the degree (map<key=degree, value=set of instance>). If we call:
+
+```graql
+compute centrality; using degree;
+```
+
+On the graph below we expect the degrees of the people to be:
+
+* **Barbara Herchelroth**: 3 - two marriages and a child
```

Comment:
Simply display the raw output from the console will boost familiarity and makes it easier to read, in my opinion:
```
$ ./graql console
>>> compute centrality of person; using degree;
{
  3: [Barbara Herchelroth],
  2: [Jacob Young, Mary Young],
  1: [John Newman]
}

Reply:
The raw output only contains degrees and a bunch of ids ...